ServerCore/Pages/Teams/TeamListBase.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    public class TeamListBase : EventSpecificPageModel
+    {
+        public TeamListBase(PuzzleServerContext serverContext, UserManager<IdentityUser> userManager) : base(serverContext, userManager)
+        {
+        }
+
+        /// <summary>
+        /// All teams
+        /// </summary>
+        public Dictionary<Team, int> Teams { get; set; }
+
+        public int PlayerCount { get; set; }
```

Comment:
It might help to have a comment here that this is players across all teams - on first read it looks like it could be for just one team, and it was only after jumping down to the function that I realized it was for everyone (although admittedly it wouldn't make sense to have it here for just one team)